pkg/state/models.go

```diff
@@ -14,6 +14,7 @@ type State interface {
 	CurrentKustomizeOverlay(filename string) (string, bool)
 	CurrentHelmValues() string
 	CurrentHelmValuesDefaults() string
+	CurrentReleaseName() string
```

Comment:
I don't think we need to continue adding "CurrentXXXX" methods to the `State` interface -- you should be able to just use `.Versioned()` and then access the release name directly?